usaspending_api/api_contracts/contracts/awards/idvs/Activity.md

```diff
@@ -17,7 +17,9 @@ This endpoint is used to power the IDV (Indefinite Delivery Vehicle) Activity vi
         + `page` (optional, number)
             The page of results to return based on the limit.
             + Default: 1
-
+        + `hide_edge_cases` (optional, boolean)
+            Choose whether or not to hide awards that have no/negative obligated amounts and/or no/negative awarded amounts and/or no end date
+            + Default: False
```

Comment:
Not sure how much capitalization matters but the API Blueprint [documentation](https://apiblueprint.org/documentation/mson/specification.html#211-primitive-types) uses all lowercase for booleans.

Reply:
I'll change it just to be safe